src/components/layout/Box/index.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import "./styles.scss";
+
+export const Box = ({
+  gap,
+  children,
+  customValue,
+  addlClassName,
+}: (
+  | { gap: "xs" | "sm" | "md" | "lg" | "xl" | "xxl"; customValue?: undefined }
+  | { gap: "custom"; customValue: string }
+) & { children: React.ReactElement; addlClassName?: string }) => {
```

Comment:
I think `l` is a typo. It shold be `addClassName`?

Reply:
It's actually "additional classes". We went with that convention in the SDS ([example](https://github.com/stellar/stellar-design-system/blob/sds-v3/%40stellar/design-system/src/components/Link/index.tsx#L27)). Thanks for checking!